README.md

```diff
@@ -12,69 +12,68 @@ There are some restrictions on the data structure of the Neo4j backend to be ful
 
 To run the web server directly:
 
-Create a virtual Environment and activate.
+#### Create a virtual Environment and activate.
 
     cd <PLATER-ROOT>
     python<version> -m venv venv
     source venv/bin/activate
     
- Install dependencies
+#### Install dependencies
     
-    pip install -r requirements.txt
+    pip install -r PLATER/requirements.txt
     
  
- Configure NEO4J Host settings
- 
-    export NEO4J_HOST=localhost
-    export NEO4J_HTTP_PORT=7474
-    export NEO4J_USERNAME=neo4j
-    export NEO4J_PASSOWORD=neo4j
-    export WEB_HOST=0.0.0.0 # <Ip to use Uvicorn web server host>
-    export WEB_PORT=8080 <PORT for the web server >
-    export PLATER_TITLE="My Plater"
+#### Configure PLATER settings
+   
+   Populate `.env-template` file with settings and save as `.env` in repo root dir.
+   
+   ```bash   
+    WEB_HOST=0.0.0.0
+    WEB_PORT=8080
+    NEO4J_HOST=neo4j
+    NEO4J_USERNAME=neo4j
+    NEO4J_PASSWORD=<change_me>    
+    NEO4J_HTTP_PORT=7474
+    PLATER_TITLE='Plater'
+    PLATER_VERSION='1.0.0'
+    
+    # Optional config for integration with an Automat server. 
```

Comment:
These Automat-related configuration bits can be removed.

Reply:
I've removed the PLATER_SERVICE address and AUTOMAT_HOST .